apps/site/lib/site_web/views/cms_view.ex

```diff
@@ -4,7 +4,7 @@ defmodule SiteWeb.CMSView do
   """
   use SiteWeb, :view
 
-  import SiteWeb.TimeHelpers
+  import SiteWeb.TimeHelpers, except: [format_time: 1]
```

Comment:
This seems potentially confusing as it's unclear which helpers we _do_ use, just from looking at this line. I think using `only` might be better here.